### PR TITLE
Nodejs: new targets

### PR DESCRIPTION
**Pull request by Avgor46**

### CODE REVIEW

File: projects/nodejs/sydr_main.cc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include <cstdint>
```
Comment (SweetVishnya) [inline projects/nodejs/sydr_main.cc:1]: Now `/opt/StandaloneFuzzTargetMain.c` is in docker.

File: projects/nodejs/env-sydr-fuzz.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+################################################################################
+[sydr]
+args = "-l debug -j2"
+target = "/load_env_sydr @@"
+jobs = 2
+
+[libfuzzer]
+path = "/load_env_fuzzer"
+args = "-rss_limit_mb=8192 -timeout=30 -jobs=1000 -workers=4 /corpus" # corpus is too large now, merge does not end
```
Comment (SweetVishnya) [inline projects/nodejs/env-sydr-fuzz.toml:23]: remove comment?

File: projects/nodejs/url-sydr-fuzz-afl++.toml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#
+################################################################################
+[sydr]
+args = "-l debug -j2"
+target = "/load_url_sydr @@"
+jobs = 2
+
+[aflplusplus]
+target = "/load_url_afl @@"
+args = "-t 60000 -i /url_corpus -x /js.dict"
```
Comment (SweetVishnya) [inline projects/nodejs/url-sydr-fuzz-afl++.toml:23]: does js.dict help for url fuzzing?

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/nodejs/sydr_main.cc (hunk 1 above)
Comment (Avgor46) [reply projects/nodejs/sydr_main.cc:1]: I need "extern \"C\"" in  `/opt/StandaloneFuzzTargetMain.c`  to compile target

File: projects/nodejs/url-sydr-fuzz-afl++.toml (hunk 3 above)
Comment (Avgor46) [reply projects/nodejs/url-sydr-fuzz-afl++.toml]: removed

File: projects/nodejs/env-sydr-fuzz.toml (hunk 2 above)
Comment (Avgor46) [reply projects/nodejs/env-sydr-fuzz.toml]: removed

File: projects/nodejs/sydr_main.cc (hunk 1 above)
Comment (Avgor46) [reply projects/nodejs/sydr_main.cc]: reworked: building StandaloneFuzzTargetMain.c into object file without patch

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**